### DISCUSSION TITLE
I've found a way to use Linux in school

### DISCUSSION THREAD

**Discussion by purpleguy5728** [TechItAllServices/TechItAllServices-Main#11] [General]
[https://linuxcontainers.org/incus/try-it/?id=584daa7f-c09f-44b3-a650-6e42e844555d#introduction](url)

**Comment by purpleguy5728** [comment]
@TechItAllSA

**Reply by TechItAllSA** [reply]
?

**Reply by purpleguy5728** [reply]
so what's the correct link for it

**Reply by TechItAllSA** [reply]
later, I need to finish this.

**Reply by purpleguy5728** [reply]
ok

**Comment by purpleguy5728** [comment]
could i get the web link for home
